contracts/RiskManagerV1.sol

```diff
@@ -807,10 +809,25 @@ contract RiskManagerV1 is IRiskManagerV1, Auctioneer, Ownable {
         view
         returns (bool)
     {
-        uint256 state = deposit.currentState();
+        DepositState state = uintToState(deposit.currentState());
+
+        return (state == DepositState.LIQUIDATION_IN_PROGRESS ||
+            state == DepositState.FRAUD_LIQUIDATION_IN_PROGRESS);
+    }
 
-        return (state == DEPOSIT_LIQUIDATION_IN_PROGRESS_STATE ||
-            state == DEPOSIT_FRAUD_LIQUIDATION_IN_PROGRESS_STATE);
+    /// @dev To learn more about deposit states see
+    /// https://github.com/keep-network/tbtc/blob/v1.1.0/solidity/contracts/deposit/DepositStates.sol
+    function uintToState(uint256 state) internal pure returns (DepositState) {
+        if (state == 9) {
```

Comment:
It feels to me that we still have the same issue with hardcoding states with these `if` statements. We _probably_ can work on adding `DepositStates.sol` contract and calling to check a deposit state but, `tbtc` doesn't go anywhere and these states won't change. Idk, maybe we can leave as it is currently implemented?

Reply:
I have the same feeling, but wanted to check how it would look if we replaced consts with enums.
@pdyraga WDYT?

Reply:
Yeah, I do agree with @dimpar, will post some more thoughts in the issue.